sdk/python/feast/permissions/client/arrow_flight_auth_interceptor.py

```diff
@@ -1,8 +1,13 @@
+import os
+
 import pyarrow.flight as fl
 
 from feast.permissions.auth.auth_type import AuthType
 from feast.permissions.auth_model import AuthConfig
-from feast.permissions.client.auth_client_manager_factory import get_auth_token
+from feast.permissions.client.auth_client_manager_factory import (
+    create_skip_auth_token,
+    get_auth_token,
+)
 
```

Comment:
Why we are doing this inside client

Reply:
I mean create_skip_auth_token,

Reply:
I see the server will behave as a client

Reply:
This is the code to create a token with the value that the server recognizes and then doesn't run the normal authorization flow